bin/cli

```diff
@@ -135,17 +131,17 @@ const getChosenKitDependency = () => {
       stdio: 'inherit',
       cwd: installDirectory
     }, console.log)
-  } else if (command === 'init') {
+  } else if (argv.command === 'init') {
     // `init` is stage two of the install process (see above), it should be
     // called by `create` with the correct arguments.
 
-    if (additionalOptions[0] !== '--') {
+    if (process.argv[3] !== '--') {
       usage()
       process.exitCode = 2
       return
     }
 
-    const installDirectory = additionalOptions[1]
+    const installDirectory = getInstallLocation()
```

Comment:
`create` needs to read args from `process.argv`, because `parse` can't handle the `--`

```suggestion
    const installDirectory = process.argv[4]
```